types/netgear/netgear-tests.ts

```diff
@@ -0,0 +1,255 @@
+import Netgear from 'netgear';
```

Comment:
```suggestion
import Netgear = require('netgear');
```
https://github.com/gruijter/netgear.js#quickstart